lib/src/main/java/uk/gov/di/ipv/core/library/service/AccessTokenService.java

```diff
@@ -64,4 +73,23 @@ public void persistAccessToken(AccessTokenResponse tokenResponse, String ipvSess
         accessTokenItem.setIpvSessionId(ipvSessionId);
         dataStore.create(accessTokenItem);
     }
+
+    public ValidationResult<ErrorObject> extractJwt(String requestBody) {
+
+        Map<String, String> stringMap = RequestHelper.parseRequestBody(requestBody);
+
+        try {
+            SignedJWT str = SignedJWT.parse(String.valueOf(stringMap.get("client_assertion")));
```

Comment:
Not sure what str is supposed to stand for? I'm guessing String maybe? If so then could we rename this to be something like jwt or clientJwt

Reply:
renamed to clientJwt